### PR TITLE
Confirm selected or first

### PR DESCRIPTION
**Pull request by artagnon**
Creating a separate PR as requested (based on delete-dir-component or `master`).

### CODE REVIEW

**Comment by artagnon** [comment]
Rebased.

File: CHANGELOG.MD [hunk 1]
```diff
@@ -16,6 +16,8 @@
 * Scroll cursor in input to the end of the text when triggering autocomplete.
 * Add `advanced-open-file:delete-path-component` command for easily navigating
   directory hierarchies. (artagnon)
+* Add `advanced-open-file:confirm-selected-or-first` command for easier opening
```
Comment (Osmose) [inline CHANGELOG.MD]: This should be under a new pending version number, I'll move it while rebasing.

Comment (Osmose) [inline CHANGELOG.MD]: This should be under a new pending version (which I added a short while ago), I'll rebase and then move it.

**Comment by Osmose** [comment]
Nice work, thanks for the fix!

**Pull request merged**